### ISSUE TITLE
Beyhad Bishvilha - No password in login page

### ISSUE THREAD

**Issue by gczobel** [eshaham/israeli-bank-scrapers#690]
Hi
The Beyhad Bishvilha is no longer working.

`Error: failed to find element matching selector "#loginId"`

 @esakal There is a way to fix that? I don't see the password option anymore...

**Comment by esakal** [comment]
Yes this is unfortunate that they don't provide a way to choose between OTP and password like many other institutes, Scrapping them really made my life easier as Beyahad Bishvilha UX is annoying to manually copy transactions from.

There are some alternatives to introduce support for OTP in (any) scrapers but since these library is in maintain mode and not under active development I don't believe that I'll add something like that.

Sorry :/
